src/App.js

```diff
@@ -51,50 +52,52 @@ class App extends Component {
                     { ...props }
                     snippets={{ ...snippets.header, langCode: snippets.langCode }} />);
               } } />
-            <Route
-              exact
-              path="/"
-              component={ (props) => {
-                return (
-                  <HomePage
-                    { ...props }
-                    snippets={{ ...snippets.homePage, langCode: snippets.langCode }} />);
-              } } />
-            <Route
-              path="/about"
-              component={ (props) => {
-                return (
-                  <AboutPage
-                    { ...props }
-                    snippets={{ ...snippets.aboutPage, langCode: snippets.langCode }} />);
-              } } />
-            <Route
-              path="/visit/:clientId/:visitId"
-              component={ (props) => {
-                return (
-                  <VisitPage
-                    { ...props }
-                    confirmer = { confirmer }
-                    snippets  = {{ ...snippets.visitPage, langCode: snippets.langCode }} />);
-              } } />
-            <Route
-              path="/visit/load"
-              component={ (props) => {
-                return (
-                  <VisitPage
-                    { ...props }
-                    confirmer = { confirmer }
-                    snippets  = {{ ...snippets.visitPage, langCode: snippets.langCode }} />);
-              } } />
-            <Route
-              path="/load"
-              component={ (props) => {
-                return (
-                  <VisitPage
-                    { ...props }
-                    confirmer = { confirmer }
-                    snippets  = {{ ...snippets.visitPage, langCode: snippets.langCode }} />);
-              } } />
+            <Switch>
+              <Route
+                exact
+                path="/"
+                component={ (props) => {
+                  return (
+                    <HomePage
+                      { ...props }
+                      snippets={{ ...snippets.homePage, langCode: snippets.langCode }} />);
+                } } />
+              <Route
+                path="/about"
+                component={ (props) => {
+                  return (
+                    <AboutPage
+                      { ...props }
+                      snippets={{ ...snippets.aboutPage, langCode: snippets.langCode }} />);
+                } } />
+              <Route
+                path="/visit/:clientId/:visitId"
+                component={ (props) => {
+                  return (
+                    <VisitPage
+                      { ...props }
+                      confirmer = { confirmer }
+                      snippets  = {{ ...snippets.visitPage, langCode: snippets.langCode }} />);
+                } } />
+              <Route
+                path="/visit/load"
```

Comment:
Can this ever be reached if it's not put above the previous route?

Reply:
As long as `"/visit/load"` doesn't match `"/visit/:a/:b"`, then that's fine! You were right about the conflict with Ethan's PR, but once that's worked out, this looks fine!

Reply:
Come to think of it, I haven't really tested it thoroughly, but I'll do more of that once the conflicts are resolved.